### PR TITLE
Update lightreaver.lua

### PR DESCRIPTION
**Pull request by Ninjistix**
adjusted proc rate

### CODE REVIEW

File: scripts/globals/items/lightreaver.lua [hunk 1]
```diff
@@ -18,7 +18,7 @@ function onAdditionalEffect(player,target,damage)
 
     -- Now continue with normal resistance and random number checks..
     local procRate = 3;
-    if (procRate >= math.random(1,100) or applyResistanceAddEffect(player,target,ELE_DARK,0) <= 0.5) then
+    if (procRate <= math.random(1,100) or applyResistanceAddEffect(player,target,ELE_DARK,0) <= 0.5) then
```
Comment (TeoTwawki) [inline scripts/globals/items/lightreaver.lua]: if 3 is less than or equal to 1to100? it's going to be less than very often.

**Comment by teschnei** [comment]
it has a 97% proc chance?

**Comment by R3P0FFXI** [comment]
@TeoTwawki @teschnei There is actually no data on BG wiki or FFXIwiki regarding the proc rate of this. The additional Effect is Death so this is most likely a "guess"

**Comment by teschnei** [comment]
you are going to guess that it procs 97% of the time?

**Comment by Ninjistix** [comment]
this was reported on the server "Lightreaver Great axe seems to have 95%-99% proc rate with death on any mob which isn't Impossible to gauge."

File: scripts/globals/items/lightreaver.lua (hunk 1 above)
Comment (TeoTwawki) [reply scripts/globals/items/lightreaver.lua]: @teschnei actually, look at the rest githubs not showing us
I derped

**Comment by TeoTwawki** [comment]
reversed my checks withotu reversing my signage
github didn't display the lined beneath sicne he didn't change them
he's right

**Comment by TeoTwawki** [comment]
``lua
    -- Now continue with normal resistance and random number checks..
    local procRate = 3;
    if (procRate <= math.random(1,100) or applyResistanceAddEffect(player,target,ELE_DARK,0) <= 0.5) then
        -- If random number beats procRate OR if effect is resisted, return all zero
        return 0,0,0;
    else
        -- It's clobberin time!
        target:setHP(0)
        return SUBEFFECT_DEATH, msgBasic.ADD_EFFECT_STATUS, EFFECT_KO;
    end
end;
```

**Comment by TeoTwawki** [comment]
@R3P0FFXI 
thats not the check for it going off
its the check for it NOT going off

**Comment by teschnei** [comment]
takeaway: teo writes really confusing code sometimes

**Pull request merged**